public/js/schedule.js

```diff
@@ -123,7 +127,11 @@
             relatedEntries.forEach(element => element.classList.remove('highlighted'));
 
             if (tooltip) {
-                tooltip.classList.remove('is-left', 'is-bottom');
+                if (this.activeTimeout) {
+                    clearTimeout(this.activeTimeout);
+                }
+
+                tooltip.classList.remove('is-left', 'is-bottom', 'entry-is-hovered');
```

Comment:
can be in the else case, right? I mean, the flyout does not show up if the timeout is cancelled.